indexer_util/indexer_util/indexer_util.py

```diff
@@ -80,20 +109,31 @@ def _wait_elasticsearch_healthy(es):
             print('Elasticsearch took %d seconds to come up.' %
                   (time.time() - start))
             break
-        except ConnectionError:
+        except ConnectionError as e:
             print('Elasticsearch not up yet, will try again.')
+            print(e)
             time.sleep(1)
     else:
         raise EnvironmentError("Elasticsearch failed to start.")
     logging.getLogger("elasticsearch").setLevel(logging.INFO)
 
 
-def get_es_client(elasticsearch_url):
-    # Retry flags needed for large datasets.
-    es = Elasticsearch([elasticsearch_url],
-                       retry_on_timeout=True,
-                       max_retries=10,
-                       timeout=30)
+def get_es_client(elasticsearch_url, deploy_local):
```

Comment:
Please add a function comment about what "deploy_local" means.
It would be nice if instead we had a field `provider` which accepted options ['local', 'gke'].
I actually think that the default would then be `local` (or we could have no default).